Detectors/util/ecal_layer_stack.py

```diff
@@ -305,6 +327,12 @@ def tungsten(t) :
     def pcb() :
         return Layer('PCB',1.666)
 
+    def pcb_unpacked():
+        return [
+            Layer('Cu', 0.170),
+            Layer('FR4', 1.030)
+        ]
+
```

Comment:
replace `pcb` with `pcb_unpacked` and drop the `PCB` material from the material mapping to avoid confusion.